sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/BasePrediction.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.Language.Conversations.Models
+{
+    public partial class BasePrediction
```

Comment:
Why was this moved out of "Generated"? In general, we don't want to take manual ownership of these files.

Reply:
I also don't see any significant difference - just whitespace changes.

Reply:
Yeah I wanted to tell you about this one. There is a mismatch between the swagger and the actual response in the field named `projectType`. it's name in the swagger is `projectKind` while the actual response has `projectType`. So I had to change it based on the [renaming guidelines](https://github.com/chamons/autorest.csharp#customize-serializationdeserialization-methods)

Reply:
Taking over the de/serialization, though, is very risky. Instead, in this case, you can use a transform. Adding the following to autorest.md would be best, and make sure to file a bug assigned to @ChongTang so he knows.

In autorest.md, look for the "transform" block and, at the bottom of the first one, add:

```yaml
- from: swagger-document
  where: $.definitions.BasePrediction
  transform: |
    $.discriminator = "projectType";
    $.required = [ "projectType" ];
    $.projectType = $.projectKind;
    delete $.projectKind;
```